MAUI/ListView/viewappearance.md

```diff
@@ -0,0 +1,1208 @@
+                    <DataTemplate x:Name="ItemTemplate"  x:Key="ItemTemplate" >
+                        <ViewCell>
+                            <ViewCell.View>
+                                    <Image Source="{Binding ContactImage}" />
+                            </ViewCell.View>
+                        </ViewCell>
+                    </DataTemplate>
+                </syncfusion:SfListView.ItemTemplate>
+            </syncfusion:SfListView>
+        </Grid>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        list.BindingContext = new ContactsViewModel();
+
+        var grid = new Grid();
+        var label = new Label();
+        label.Text = "Tap image to expand";
+        var grid1 = new Grid();
+        var image = new Image();
+        image.SetBinding(Image.SourceProperty, new Binding("ContactImage"));
+        var label1 = new Label();
+        label.SetBinding(Label.TextProperty, new Binding("ContactName"));
+        var label2 = new Label();
+        label1.SetBinding(Label.TextProperty, new Binding("ContactNumber"));
+        grid1.Children.Add(image);
+        grid1.Children.Add(label1);
+        grid1.Children.Add(label2);
+
+        var listView = new SfListView();
+        ContactsViewModel contactsViewModel = new ContactsViewModel();
+        listView.ItemsSource = contactsViewModel.ContactsInfo;
+        listView.ItemSize = 70;
+        listView.ItemTapped += ListView_ItemTapped;
+        listView.ItemTemplate = new DataTemplate(() =>
+        {
+            var viewCell = new ViewCell();
+            var image1 = new Image();
+            image1.SetBinding(Image.SourceProperty, new Binding("ContactImage"));
+            viewCell.View = image1;
+            return viewCell;
+        });
+
+        grid.Children.Add(label);
+        grid.Children.Add(grid1);
+        grid.Children.Add(listView);
+
+    }
+
+    private void list_ItemTapped(object sender, Syncfusion.ListView.XForms.ItemTappedEventArgs e)
+    {
+        GridView.BindingContext = e.ItemData;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Horizontal list inside vertical list
+
+The `SfListView` allows you to layout the items in horizontal list inside the vertical list. You can load the nested `SfListView` by customizing the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) of outer `SfListView`. 
+
+You should define the size for each inner `SfListView` or set the [AutoFitMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_AutoFitModeProperty) of inner SfListView as `Height`, and define the [ItemSize](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemSize) for outer `SfListView`.
```

Comment:
You should either define the size for each inner `SfListView` or set the [AutoFitMode](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_AutoFitModeProperty) of the inner SfListView to `Height`, and define the [ItemSize](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemSize) for outer `SfListView`.

Reply:
I have modified the line as per the concerns provided.